website/docs/d/kvstore_instance_classes.html.markdown

```diff
@@ -0,0 +1,50 @@
+---
+layout: "alicloud"
+page_title: "Alicloud: alicloud_kvstore_instance_classes"
+sidebar_current: "docs-alicloud-datasource-kvstore-instance-classes"
+description: |-
+    Provides a list of KVStore instacne classes info.
+---
+
+# alicloud\_kvstore\_instances\_classes
+
+This data source provides the KVStore instance classes resource available info of Alibaba Cloud.
+
+-> **NOTE:** Available in v1.48.0+
+
+## Example Usage
+
+```tf
+data "alicloud_kvstore_instance_classes" "resources" {
```

Comment:
zone_id is required ,this is listed

Reply:
zone_id is specified in the interface of kvstore